### ISSUE TITLE
CIFAR-10 data import problem 

### ISSUE THREAD

**Issue by hexakarthikeyan** [Hvass-Labs/TensorFlow-Tutorials#1]
Hi  
     I was trying to import CIFAR-10 dataset in my machine but an error raised saying that no module named CIFAR-10. How am i suppose to import CIFAR-10 dataset in python. i am working in python 2.7 environment .

**Comment by papachristoumarios** [comment]
Hello, make sure the source file `cifar10.py` is at the same directory as your notebook. Then you won't have problems importing it.

**Comment by Hvass-Labs** [comment]
@papachristoumarios Thanks for answering this!

I'll add some installation instructions to the README soon.

**Comment by samadak** [comment]
Hi.
My problem is how to add cifar 10.py to my jupyter notebook
because jupyter placed in virtual environment.
please assist me to understand how to do it?

**Comment by Hvass-Labs** [comment]
Please see the installation instructions. It is best to download / clone the entire GitHub repository for these tutorials, so you have all the files you need.

**Comment by samadak** [comment]
Thank you very much
I have no idea about fine tune CIFAR 10 NET for my dataset 
I know that I should change the learning rate but i don't know how to do it?
I was wondering if you could help me
